src/index.js

```diff
@@ -13,7 +13,7 @@ const trialDivision = number => {
 			if (number % divisor === 0) {
 				isPrime = false;
 			}
-			divisor++;
+			divisor += 2;
```

Comment:
Podrias usar un for y hacer un genrator parecido a este
```js
function* nextDivisor() {
    yield 2;
   let divisor = 3;
    while(true){
        yied divisor;
       divisor++;
    }
}
```
jaja creo que es sobre ingería xD

Reply:
Si quiere meter mas eurística, si squere es entero, ya sabes que no es un número primo.